unittests/scans/csaf/csaf_without_vulns.json

```diff
@@ -0,0 +1,42 @@
+{
+    "document": {
+      "category": "csaf_security_advisory",
+      "csaf_version": "2.0",
+      "publisher": {
+        "category": "user",
+        "name": "test_publisher",
+        "namespace": "http://www.foo.de"
+      },
+      "title": "A very serious vulnerability for testing",
+      "tracking": {
+        "current_release_date": "2022-06-01T10:00:00.000Z",
+        "id": "Example Vulnerability - Testing Company",
+        "initial_release_date": "2022-06-01T10:00:00.000Z",
+        "revision_history": [
+          {
+            "date": "2022-06-01T10:00:00.000Z",
+            "number": "1.0.0",
+            "summary": "first version"
+          }
+        ],
+        "status": "final",
+        "version": "1.0.0",
+        "generator": {
+          "date": "2022-06-16T16:14:14.778Z",
+          "engine": {
+            "version": "1.14.0",
+            "name": "Secvisogram"
+          }
+        }
+      }
+    },
+    "vulnerabilities": [],
+    "product_tree": {
+      "full_product_names": [
+        {
+          "product_id": "CSAFPID-1337",
+          "name": "A very cool product"
+        }
+      ]
+    }

```

Comment:
Why not remove these lines completely? If you change the `/document/category` to `csaf_base` the document itself would be valid...